app/Http/Controllers/MainController.php

```diff
@@ -38,16 +38,25 @@ public function place(){
         return view('View_place', ['cities_names' => $cities_names,'stay'=> $stay,'place_name'=> $place_name]);
 
     }
-    public function getPlace(Request $request)
+    public function getPlaces(Request $request)
     {
         $id_req = $request->city_id;
         $places = Place::all()->where('id_city', $id_req);
         return array_map(function($place){
             return array ('id' => $place->id, 'text' => $place->address);
         },$places->all());
     }
+    public function getPlaces2(Request $request)
+    {
+        $id_req = $request->place_id;
+
+        $places = Place::all()->where('id_city', $id_req);
+        return array_map(function($place){
+            return array ('id' => $place->id, 'text' => $place->address);
+        },$places->all());
+    }
 
-    public function getPlaceLocation(Request $request){
+    public function getSpaces(Request $request){
```

Comment:
```
$name_places = NamePlace::all()->where('place_id', $id_req);
```
Can we have few plases with the same ID ?

Reply:
Yes, they all have the same id of the city. There is array of places.

Reply:
We should rename this field in future (
Plz, add the comment in this part of code too.